### PR TITLE
KIT-165 front

### PR DESCRIPTION
**Pull request by emmirose**
HTTP request to get all todos

### CODE REVIEW

File: src/app/components/todolist/todolist.component.html
Comment (emericb) [inline src/app/components/todolist/todolist.component.html:1]: remove dead code

**Review by emericb — CHANGES_REQUESTED** [review]

**Review by emericb — APPROVED** [review]

File: src/app/pages/todolists/todolists.component.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export class TodolistsComponent {
+  todolists: ToDoList[] = [];
+  todolist!: ToDoList;
+
+  constructor(private todolistsService: TodolistsService) {}
+
+  ngOnInit() {
+    this.todolistsService.getLists().subscribe((data) => {
+      this.todolists = data;
+      console.log(this.todolists);
```
Comment (Benja-san) [inline src/app/pages/todolists/todolists.component.ts:19]: remove console logs

File: src/app/pages/todolists/todolists.component.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { ToDoList } from 'src/app/models/ToDoList';
+
+@Component({
+  selector: 'app-todolists',
+  templateUrl: './todolists.component.html',
+  styleUrls: ['./todolists.component.scss'],
+})
+export class TodolistsComponent {
+  todolists: ToDoList[] = [];
+  todolist!: ToDoList;
```
Comment (Benja-san) [inline src/app/pages/todolists/todolists.component.ts:12]: what does this property stand for ?

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/pages/todolists/todolists.component.ts (hunk 2 above)
Comment (emmirose) [reply src/app/pages/todolists/todolists.component.ts]: Not necessary anymore !

**Review by emericb — DISMISSED** [review]

**Review by Benja-san — DISMISSED** [review]

**Review by emericb — APPROVED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**